playbooks/roles/cloudwatch/templates/elasticsearch_awslogs.conf.j2

```diff
@@ -0,0 +1,8 @@
+[general]
+state_file = /var/awslogs/state/agent-state
+
+[elasticsearch-log]
+file = /var/log/elasticsearch/{{ elasticsearch_name }}.log
+log_group_name = {{ cloudwatch_log_group }}
+log_stream_name = {{ elasticsearch_name }}
```

Comment:
I'm not sure if that's what you mean, but GroupTag is used for elasticsearch cluster discovery (so that nodes only join within their own environment). Since nginx doesn't cluster there's no need for GroupTag.

The stream name is a bit redundant, `elasticsearch-preview-preview` inside the `preview-preview` group. But since it's also the cluster name, I think it makes sense.